### ISSUE TITLE
Et tu, WFN?

### ISSUE THREAD

**Issue by African-Networks** [wokhansoft/WFN#8]
![wfn](https://user-images.githubusercontent.com/29213984/27247311-e0c8b970-52e6-11e7-9127-99b287ec8226.png)

**Comment by AtlasHackert** [comment]
Yes, WFN also accesses the internet. In this particular case, it's the Notifier doing name resolution (which is something that's on my does-it-really-need-it to-check list). WFN.exe's map functionality also needs internet access to obtain your current public (internet) IP.

This is in fact a very powerful 'feature': WFN doesn't sneakily add itself to your firewall rules. So yes, you'll get these "silly" notifications, but it's just WFN playing it honest.

**Comment by African-Networks** [comment]
Thanks for the clarification, but I would still oppose the "sneakily" part. This screenshot was taken from virtual machine, but I have never seen such popup on a "real machine", where the only DNS related entry in firewall belongs to svchost.exe (Custom Rule - DNS Client[Dnscache] [T:192.168.1.1] [R:53]). So there is suspicion of WFN is also "calling home".

**Comment by AtlasHackert** [comment]
In virtual networks things work slightly differently, and that may affect the way WFN sees connection-attempts. Pop-ups like this for DNS for example also turn up when you use VPN-software.

At any rate, the pop-up clearly states WFN is trying to connect to your gateway (192.168.1.1), so it's most definitely not calling home in your screenshot. You can also verify this by reading the source code, or using network sniffers.

Still, I should probably remove the DNS-query, as it appears it's not needed anyway.
